docker-compose.yml

```diff
@@ -1,14 +1,27 @@
 version: "3"
 services:
-  hmpps-interventions-service:
-    build:
-      context: .
-    network_mode: "host"
-    container_name: hmpps-interventions-service
+  hmpps-auth:
+    image: quay.io/hmpps/hmpps-auth:latest
```

Comment:
Very generic question, sorry. Is it safe to assume `latest` is the most recent build on the `main` branch?

Reply:
that's right